### ISSUE TITLE
Youtube video ads

### ISSUE THREAD

**Issue by peterlaws** [andryou/scriptsafe#297]
Not an issue as such.

Has Google found a way to bypass adblockers/etc?

Sites are getting vigilant against blockers, some just know things are blocked.

For months now, video ads are increasingly playing.

I only have SzafeScript and AdMuncher installed.

**Comment by andryou** [comment]
I may suggest looking into uBlock Origin: https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm

I am unsure if AdMuncher is still being maintained/developed, so give uBlock Origin a try with AdMuncher temporarily disabled.

**Comment by peterlaws** [comment]
Since adding that:

![fireshot-278](https://user-images.githubusercontent.com/9319293/33800274-662af1f0-dd34-11e7-883f-60e74ea2d314.png)

**Comment by andryou** [comment]
Yup! It's safe to ignore that, it's just a notification that uBlock and ScriptSafe are both trying to redirect an unwanted image to a transparent gif/png.
